### ISSUE TITLE
Support questionnaires and activities

### ISSUE THREAD

**Issue by blootsvoets** [RADAR-CNS/ManagementPortal#38]
It would be useful to have some way of entering structured exercises and questionnaires to the management portal. Perhaps check with @kerzmaximilian, @fnobilia and @hesamsagha what properties would be useful there.

**Comment by fnobilia** [comment]
I agree. Having a centralise repository for configuration files simplifies the interaction and improves the maintainability. For example a starting point may be storing this [JSON Schema](https://github.com/RADAR-CNS/RADAR-Schemas/blob/phq8/questionnaire/specification/PHQ8_config.json) there and providing a way to configure it.

It looks like we may reduce our dependences on Firebase configuration using the management portal.

I would suggest a low priority for this thing. The Firebase implementation is already in place.

**Comment by fnobilia** [comment]
In order to be consistent across all projects, the `JSON` file should be converted into `YAML`.

We can create `AVRO` entity if needed to define single variable. Using the [YamlConfigLoader](https://github.com/RADAR-CNS/RADAR-Commons/blob/dev/src/main/java/org/radarcns/config/YamlConfigLoader.java) of `RADAR-Common` we can validate the `YAML` file against the `AVRO` schemas while parsing.

In the end, we can expose a `JSON` end point to retrieve the configuration file for the `Front-end` ecosystem.

**Comment by speelen** [comment]
The PHQ8 [JSON file](https://github.com/RADAR-CNS/RADAR-Schemas/blob/phq8/questionnaire/specification/PHQ8_config.json) lacks question 5 "Poor appetite or over eating". Furthermore, I would start labelling the first question as **PHQ8-1**, just to be in line with the human readable questionnaire itself.
